src/api-requests/get-industry-events.ts

```diff
@@ -116,3 +116,21 @@ export const getIndustryEventBySlugFromAPI = async (
 
     return data.industryEvents.length > 0 ? data.industryEvents[0] : null;
 };
+
+export const getDraftEventFromAPI = async (
+    client: UnderlyingClient<'ApolloREST'>,
+    calculatedSlug: string
+): Promise<IndustryEvent | null> => {
+    calculatedSlug = '"' + calculatedSlug + '"';
```

Comment:
what is it you're trying to do here?

Reply:
have to construct Graphql args like this - add quotes to the calculated slug.

Reply:
ok, i was just confused by the single quote, double quote, single quote pattern. I would say just put quotes around your `calculatedSlug` template literal on line 127:

`"${calculatedSlug}"`